doc/19_Development_Tools_and_Details/09_Cache/03_Full_Page_Cache.md

```diff
@@ -41,18 +41,22 @@ pimcore:
 | ------ | ----------- |
 | Enable | Set to true to enable to full page cache. |
 | Lifetime | You can optionally define a lifetime (in seconds) for the  full page cache. If you don't do, the cache is evicted automatically when there is a modification in the Pimcore Backend UI. If there is a lifetime the item stays in the cache even when it is changed until the TTL is over. The lifetime is useful if you have embedded some items which are not directly in the cms, like rss feeds, or twitter messages over the API. It is also highly recommended to specify a lifetime on high traffic websites so that the frontend (caches) isn't affected by changes in the admin-UI. Otherwise on every change in the admin-UI the whole output-cache is flushed, what can have drastic effects to the server environment. |
-| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters). Type one pattern in each line. |
+| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters) and different patterns should be seperated by `,` |
 | Disable Cookie | You can define an additional cookie-name which disables the cache. The cookie "pimcore_admin_sid" (used for the Pimcore admin UI) ALWAYS disables the output-cache to make editor's life easier ;-) 
 
 
 ## Disable the Full Page Cache in your Code
 Sometimes it is more useful to deactivate the full page cache directly in the code, for example when 
 it's not possible to define an exclude-regex, or for similar reasons.
 
-In this case you can use obtain the pull page cache service from the container and disable it, eg. in an action: 
+In this case you can use obtain the pull page cache service from the container and disable it, eg. in an Controller via DI: 
 ```php
 <?php
-$this->get(\Pimcore\Bundle\CoreBundle\EventListener\Frontend\FullPageCacheListener::class)->disable("Your disable reason");
+    public function portalAction(Request $request , \Pimcore\Bundle\CoreBundle\EventListener\Frontend\FullPageCacheListener $FullPageCacheListener)
+    {
+        $FullPageCacheListener->disable("Your disable reason");
+       return $this->redirect('de');
+    }
```

Comment:
some minor cs fix

```suggestion
    use Pimcore\Bundle\CoreBundle\EventListener\Frontend\FullPageCacheListener;
    
    public function portalAction(Request $request, FullPageCacheListener $fullPageCacheListener)
    {
       $fullPageCacheListener->disable("Your disable reason");
       return $this->redirect('de');
    }
```

Reply:
done

Reply:
it seems that there are two small thing from this suggestion that are missing:

- the space between Request $request and the comma should be removed
- `$fullPageCacheListener` should be camelCase  first letter in lowercase

Reply:
should be fine now